config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
@cyli I agree that secrets don't seem to be visible to plugins. I tried creating a secret and accessing it as `/run/secrets/foo` from within the plugin and had no success.